core/Network/TLS/Credentials.hs

```diff
@@ -33,9 +34,16 @@ type Credential = (CertificateChain, PrivKey)
 
 newtype Credentials = Credentials [Credential]
 
+#if MIN_VERSION_base(4,9,0)
+instance Semigroup Credentials where
+    (Credentials l1) <> (Credentials l2) = Credentials (l1 ++ l2)
```

Comment:
Parentheses in the left expression are not necessary.

Reply:
Now modified.